client/utils/notes.ts

```diff
@@ -0,0 +1,22 @@
+import { NoteManagerCore } from 'utils/notesCore';
+import { apiFetch } from 'client/utils/apiFetch';
+
+const isClient = typeof window !== 'undefined';
+
+const renderStructuredValues = (structuredValues, citationStyleKind, inlineStyleKind) =>
+	isClient
+		? apiFetch('/api/editor/citation-format', {
+				method: 'POST',
+				body: JSON.stringify({
+					structuredValues,
+					inlineStyleKind,
+					citationStyleKind,
+				}),
+		  })
+		: Promise.resolve([]);
+
+export class NoteManager extends NoteManagerCore {
```

Comment:
Were you planning to merge this with `NoteManagerCore` since we no longer have a server/client split? I'm 100% okay leaving this as-is, it's a nice abstraction.